packages/basis/src/TranscoderWorker.ts

```diff
@@ -0,0 +1,200 @@
+import { BASIS_FORMATS } from './Basis';
+import { ITranscodeResponse, TranscoderWorkerWrapper } from './TranscoderWorkerWrapper';
+import { BasisLoader } from './BasisLoader';
+
+/**
+ * Worker class for transcoding *.basis files in background threads.
+ *
+ * To enable asynchronous transcoding, you need to provide the URL to the basis_universal transcoding
+ * library.
+ *
+ * @memberof PIXI.BasisLoader
```

Comment:
Huh? Shouldn't this be `@memberof PIXI`?

Reply:
`TranscoderWorker` was designed as a "inner class" of `BasisLoader`. It doesn't serve any purpose outside of `BasisLoader`.

The inner class concept comes from Java. If you don't like it, I can move `TranscoderWorker` up into the `PIXI` namespace. The idea was just to not pollute the `PIXI` namespace.

Reply:
Okay, that's fine. Disregard my comment.